stellargraph/data/explorer.py

```diff
@@ -507,13 +507,13 @@ def run(self, nodes, n_size, n=1, seed=None):
         Performs a sampled breadth-first walk starting from the root nodes.
 
         Args:
-            nodes (list): A list of root node ids such that from each node n BFWs will be generated up to the
```

Comment:
there is no `d` depth parameter. How is the depth specified?

Reply:
The depth is inferred by the length of the list `n_size` - I'll clarify the docs